### PR TITLE
Millify Github & npm dependents count

### PR DESCRIPTION
**Pull request by vladimyr**
This is a continuation of #406 with `cheerio` being replaced by regexes :tada: 

## Before

image

---

## After

image

### CODE REVIEW

File: api/github.ts [hunk 1]
```diff
@@ -454,29 +455,24 @@ async function repoStats ({topic, owner, repo, ...restArgs}: PathArgs) {
...
+        subject,
+        status: 'invalid',
+        color: 'grey'
+      }
+    }
     return {
-      subject: type === 'PACKAGE' ? 'pkg dependents' : 'repo dependents',
-      status: parseDependents(html, type),
+      subject,
+      status: millify(count),
```
Comment (amio) [inline api/github.ts:474]: `count` might be a `NaN`, which would cause `millify` throw an error

Comment (vladimyr) [reply api/github.ts:474]: True, I'll handle it explicitly.

File: api/npm.ts [hunk 2]
```diff
@@ -163,24 +162,24 @@ const download = async (period: string, npmName: string, tag = 'latest') => {
   }
 }
 
-// https://github.com/astur/check-npm-dependents/blob/master/index.js
```
Comment (vladimyr) [inline api/npm.ts:166]: Not relevant anymore since I'm not scraping HTML but hydration data instead (JSON).

File: api/github.ts (hunk 1 above)
Comment (amio) [reply api/github.ts:474]: How I missed that block 🙈

**Pull request merged**